src/main/resources/hudson/plugins/build_timeout/impl/LikelyStuckTimeOutStrategy/help_ja.html

```diff
@@ -1,4 +1,3 @@
 <p>
-Jenkins の Executor.html#isLikelyStuck() で使用されている方法を用い、ビルドが過去の実績から推定される最大のビルド時間を超えている場合にタイムアウトします。
-<br/>
+過去の実行に比べて極端に長い時間ビルドが実行されている場合にタイムアウトと判定します。
```

Comment:
Translated from help.html: Abort the build when the job has taken many times longer than previous runs.